books/RayTracingInOneWeekend.html

```diff
@@ -1046,12 +1046,6 @@
     inline double ffmin(double a, double b) { return a <= b ? a : b; }
     inline double ffmax(double a, double b) { return a >= b ? a : b; }
 
-    inline double clamp(double x, double min, double max) {
```

Comment:
Do we replacing this with std::clamp?
If so, we need to include <algorithm>, and _maybe_ std::clamp?

Reply:
For now, we only require C++11, and `std::clamp` is C++17.

As a bonus, this also has the advantage of being more transparent to the reader.